### ISSUE TITLE
It works only once

### ISSUE THREAD

**Issue by nalevieux** [intel/openvino-plugins-ai-audacity#68]
It works only once.

Hello, You can try the track separation plugin only once. When you want to try again with another song, although you separate the tracks, the result is noise to interference or noise.

I have read a similar problem in another forum thread.
I was able to get it working just one more time, but I had to uninstall Audacity, delete the residual folders, and reinstall (Audacity and plugins folders).
After that, same thing again, I was able to get it working only once.
I currently use UVR for track separation and have no problems.
System: Windows 10 64 bit
RAM: 16GB
Micro: AMD Ryzen 5 3400G
I don't have a GPU

Greetings

**Comment by RyanMetcalfeInt8** [comment]
Hi @nalevieux,

I have seen this issue reported before, here: https://github.com/intel/openvino-plugins-ai-audacity/issues/1

I still haven't been able to reproduce this myself. Could you give some more details of the Windows 10 edition that you are using (including the language)? 

The workaround is to remove this folder before running music separation: ```%appdata%\audacity\openvino-model-cache```

Definitely not ideal, but definitely better than uninstalling entire Audacity installation as you had said above.

Thanks,
Ryan

**Comment by nalevieux** [comment]
Hi, thanks for your response.
Windows 10 Pro Version 22H2 – Build 19045.4123
Spanish Language

I tell you that I tried the solution you mentioned and it worked correctly.

Thank you so much!
A hug from Argentina.

**Comment by RyanMetcalfeInt8** [comment]
Glad it worked for you, and thanks for the info. Let me actually close this as it's a duplicate of #1 -- you may want to subscribe to that issue as I might add findings, etc. there. Thanks!
